e2e/tests/popupClickHandling-example.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { Button, Popup } from '@stardust-ui/react'
+
+export const selectors = {
+  triggerButtonId: 'trigger',
+  popupContentClass: Popup.slotClassNames.content,
+  popupContentButtonId: 'content-button',
+}
+
+const PopupClickHandlingExample = () => {
+  const [show, setShow] = React.useState(true)
+  return (
+    <Popup
+      trigger={
+        <Button id={selectors.triggerButtonId} icon="expand" onClick={() => setShow(true)} />
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/59342474-2915e180-8d0a-11e9-8b45-310a3f8ea820.png)

```suggestion
        <Button id={selectors.triggerButtonId} content='trigger' icon="expand" onClick={() => setShow(true)} />
```

nit: Can we add `content`? Otherwise this example looks like broken